src/styles/blocks/contact.scss

```
@@ -0,0 +1,80 @@
+.contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+
+  &__title {
+    text-align: center;
+    padding-bottom: 8px;
+  }
+
+  &__logo {
+    height: 19px;
+    width: 23px;
+    margin: 0 auto 61px;
+    background-image: url(../images/crown.svg);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__wrapper {
+    @media (min-width: 768px) {
+      display: flex;
+      justify-content: space-between;
+    }
+  }
+
+  &__info {
+    @media (min-width: 1000px) {
+      padding: 0 10px 0 0;
+      display: flex;
+      width: 400px;
+      justify-content: space-between;
+    }
+  }
+
+  &__communication-block {
+    padding-right: 10px;
+    text-decoration: none;
+  }
+
+  &__addres-block {
+    padding-right: 20px;
+  }
+
+  &__label {
+    font-size: 14px;
+    line-height: 27px;
+    color: #9fadc2;
+    font-family: Roboto, sans-serif;
+  }
+
+  &__phone {
+    font-size: 16px;
+    line-height: 27px;
+    color: #3e3e3e;
```

```suggestion
    color: #3e3e3e;
```
Better to create variables for font colors and font sizes. For example, you use this color three times in your cod. It would be easy to use meaningful variable than this color code